src/main/resources/config.properties

```diff
@@ -1,3 +1,3 @@
-UiPath.CLI.Version=1.0.7934.37108
+UiPath.CLI.Version=1.0.7950.8690
 UiPath.DefaultTenant=default
 UiPath.CLI.Name=UiPath.CLI
```

Comment:
I don't see this change in the PR
```
Added help text as a description that accountName must be provided for the cloud orchestrator connections.
```

Reply:
I have updated the description now , as in jenkins there is no provision of providing the help text against specific fields.